Magi/Clara, the Folk Witch/Clara, Matilde's Sister.md

```diff
@@ -107,8 +107,8 @@ Clara grew as a Folk Witch skilled in entrancing people and creature to do her b
 - Virtue: Essential Virtue (Relentless Pursuer)
 - Personal Power
 
-**Personality Trait:**   
-**Reputations:** 
+**Personality Trait:** Hatred +3, Arrogant +3   
```

Comment:
I'll add a "Hatred (towards men)" , but what pronoun are you talking about?

Reply:
Is *Baree, the wolf* a he or a she. (I guess wolves don't have the same issues with genders and pronouns as humans).

(It could matter to a SG wanting to drop some opposite-gender wolves trying to mate with Baree)